data/presets/amenity/swingerclub.json

```
@@ -0,0 +1,27 @@
+{
+    "icon": "maki-heart",
+    "fields": [
+        "{amenity/nightclub}",
+        "smoking"
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "air_conditioning",
+        "fee"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "amenity": "swingerclub"
+    },
+    "terms": [
+        "adult*",
+        "porn*",
+        "sex",
+        "strip*",
+        "swinger*"
+    ],
+    "name": "Swingerclub"
```

one last thing: wikipedia calls these establishments a [Sex Club](https://en.wikipedia.org/wiki/Sex_club). Probably worth adding as an alias. Also, I think the spelling as two words Swinger Club is the more common one in English. Maybe also adding "lifestyle club" as a search term?

```suggestion
    "terms": [
        "adult*",
        "lifestyle club",
        "porn*",
        "sex",
        "strip*",
        "swinger*"
    ],
    "name": "Swinger Club",
    "aliases": [
        "Sex Club"
    ]
```

---

Yeah, those are good points. It also got me thinking: perhaps we should consider adding the web page field to the `amenity/nightclub category`, or at least in here. Most places nowadays have their page